src/main/java/com/esir/jxc/mpc/model/event/EventArticleCreated.java

```diff
@@ -0,0 +1,22 @@
+package com.esir.jxc.mpc.model.event;
+
+import com.esir.jxc.mpc.model.Event;
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+public class EventArticleCreated {
```

Comment:
```suggestion
public class ArticleCreated {
```
➜ le prefix Event est inutile. De plus en anglais on met `Event` à la fin